src/astra_rl/core/common.py

```diff
@@ -0,0 +1,12 @@
+"""
+common.py
+Common data structures.
+"""
+
+from typing import TypeVar
+
+StateT = TypeVar("StateT")
```

Comment:
See comment below, but docstrings and/or comments would help here to understand what these are.